classes/EventRegistry.php

```diff
@@ -263,4 +265,77 @@ public function pruneTranslatedContentTemplates($templates)
             return !Str::endsWith($template->getBaseFileName(), $extensions);
         });
     }
+
+    /**
+     * Adds language suffixes to mail view files.
+     * @param  \October\Rain\Mail\Mailer $mailer
+     * @param  \Illuminate\Mail\Message $message
+     * @param  string $view
+     * @param  array $data
+     * @param  string $raw
+     * @param  string $plain
+     * @return string|null
+     */
+    public function findLocalizedMailViewContent($mailer, $message, $view, $data, $raw, $plain)
+    {
+        // There is no need to localize raw templates
+        if (!empty($raw)) {
+            return null;
+        }
+
+        $locale = !empty($data['_current_locale']) ? $data['_current_locale'] : App::getLocale();
+
+        $factory = $mailer->getViewFactory();
+
+        if (isset($view)) {
+            $view = $this->getLocalizedView($factory, $view, $locale);
+        }
+
+        if (isset($plain)) {
+            $plain = $this->getLocalizedView($factory, $plain, $locale);
+        }
+
+        $code = $view ?: $plain;
+        if (empty($code)) {
+            return null;
+        }
+
+        $plainOnly = empty($view);
+
+        $result = MailManager::instance()->addContentToMailer($message, $code, $data, $plainOnly);
+        if (!$result) {
+            // prevent the caller who fired the event from continuing to send the mail.
+            return false;
+        }
```

Comment:
This has only confused me more. Now you're saying that if the call returns false, you want to return false; but you're also saying return nothing unless the call returns falsey. Either way the return value of this method will always be falsey and thus never halt the event. Are you sure you don't just mean return the result of `MailManager::instance()->addContentToMailer()` unaltered?

Reply:
But Luke, halting the event IS done by returning false... what  am I missing here?

Reply:
but in that last commit I did reverse the logic which was fine before... 🤣

Reply:
should be good now.

Reply:
> But Luke, halting the event IS done by returning false... what am I missing here?

The question is what was I missing here 😂. I didn't read the entire line of code for the firing and completely missed the `=== false`

Reply:
Ok, I was getting worried... :relieved: